src/main/java/com/digitalasset/damlmavenplugin/Commands.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.digitalasset.damlmavenplugin;
+
+public class Commands {
+    static final String DAML = isWindows() ? "daml.cmd" : "daml";
```

Comment:
Can DAML team confirm this? The official documentation does not mention `daml.cmd`, it says launching endpoint is `daml`. So it may be a DAML issue if you need `daml.cmd`.

Reply:
On Windows the DAML installer creates the `daml.cmd` file in the `bin` folder. This launches the actual `daml.exe` of the currently installed SDK. The `bin` folder (and thus the CMD file) is put on the PATH.
*BTW: this is kind of the same on Unix systems as well: a `bin` folder is created, where is a symlink `daml` that points to the actual executable.*

As a user this is transparent. Meaning that in a terminal issuing a `daml` command works just fine. However, when `daml` is given actually `daml.cmd` will be invoked.

For other processes like Python or Java simply `daml` is not enough, suffixing with CMD is needed.

Reply:
this must be a silly question, but why can't you launch `daml` here again then?

Reply:
Starting from a terminal the shell adds the extension, so `daml` becomes `daml.cmd`. This behaviour does not occur from Python or Java.

Reply:
Ohh I see. Maybe we should signal towards the DAML team that daml.cmd may be replaced by a daml.exe in order to keep the CLI. Anyway, this change seems good.